server/routerlicious/packages/routerlicious/src/scribe/index.ts

```diff
@@ -51,6 +52,11 @@ export async function scribeCreate(config: Provider): Promise<IPartitionLambdaFa
         },
         true);
 
+    if (createCosmosDBIndexes) {
+        await scribeDeltas.createIndex({ "operation.sequenceNumber": 1 }, false);
+        await scribeDeltas.createTTLIndex({_ts:1}, mongoExpireAfterSeconds);
+    }
+
     if (mongoExpireAfterSeconds > 0) {
         await scribeDeltas.createTTLIndex(
```

Comment:
Now that we call TTLIndex above for the specific case of CosmosDB, should this be in an else statement? Or do we want to also set a TTL index called `mongoTimestamp` for CosmosDB as well?

I was wondering if we should do something like this:

```
 if (createCosmosDBIndexes) {
            await scribeDeltas.createIndex({ "operation.sequenceNumber": 1 }, false);
}

if (mongoExpireAfterSeconds > 0) {
     if (createCosmosDBIndexes) {
            await scribeDeltas.createTTLIndex({_ts:1}, mongoExpireAfterSeconds);
     } else {
            await scribeDeltas.createTTLIndex(
            {
                mongoTimestamp: 1,
            },
            mongoExpireAfterSeconds);
    }
}

```

Reply:
Same thing for Scriptorium below.